go.mod

```diff
@@ -1,53 +1,55 @@
 module github.com/canonical/microcluster
 
-go 1.22.0
+go 1.23.3
 
 require (
-	github.com/canonical/go-dqlite v1.21.0
-	github.com/canonical/lxd v0.0.0-20240403135607-df45915ce961
-	github.com/fsnotify/fsnotify v1.7.0
+	github.com/canonical/go-dqlite/v2 v2.0.0
+	github.com/canonical/lxd v0.0.0-20241209143752-5a8ba0bdd60b
+	github.com/fsnotify/fsnotify v1.8.0
 	github.com/google/renameio v1.0.1
 	github.com/gorilla/mux v1.8.1
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/spf13/cobra v1.8.0
-	golang.org/x/sys v0.18.0
+	github.com/spf13/cobra v1.8.1
+	golang.org/x/sys v0.28.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
 	github.com/Rican7/retry v0.3.1 // indirect
 	github.com/armon/go-proxyproto v0.1.0 // indirect
+	github.com/canonical/go-dqlite/v3 v3.0.0 // indirect
```

Comment:
The go-dqlite/v3 looks suspicious. Shouldn't this be v2 only? I am wondering from where it's coming from.